### PR TITLE
fix: add KUBECOST_URL to jupyter-web-app deployment

### PR DESCRIPTION
**Pull request by saffaalvi**
Resolves https://github.com/StatCan/kubeflow-manifest/issues/50

### CODE REVIEW

File: kustomize/jupyter-web-app/base/params.env [hunk 1]
```diff
@@ -5,3 +5,4 @@ prefix=jupyter
 clusterDomain=cluster.local
 userid-header=kubeflow-userid
 userid-prefix=
+KUBECOST_URL=http://0.0.0.0:9090
```
Comment (brendangadd) [inline kustomize/jupyter-web-app/base/params.env:8]: This needs to be the actual in-cluster URL.

**Review by brendangadd — CHANGES_REQUESTED** [review]

**Review by zachomedia — APPROVED** [review]
LGTM

File: kustomize/jupyter-web-app/base/deployment.yaml [hunk 2]
```diff
@@ -22,6 +22,11 @@ spec:
           value: $(userid-header)
         - name: USERID_PREFIX
           value: $(userid-prefix)
+        - name: KUBECOST_URL
+          valueFrom:
+            configMapKeyRef:
+              name: parameters
+              key: KUBECOST_URL
```
Comment (zachomedia) [inline kustomize/jupyter-web-app/base/deployment.yaml:29]: This isn't output into the config map. I'm looking further

**Review by zachomedia — CHANGES_REQUESTED** [review]

File: kustomize/jupyter-web-app/base/deployment.yaml (hunk 2 above)
Comment (zachomedia) [reply kustomize/jupyter-web-app/base/deployment.yaml:29]: Never mind, I see it gets written out

**Review by zachomedia — APPROVED** [review]

**Review by brendangadd — APPROVED** [review]

**Pull request merged**